app/controllers/exchanges/hbx_profiles_controller.rb

```diff
@@ -62,11 +62,10 @@ def employer_index
 
   def generate_invoice
 
-    @organizations= Organization.where(:id.in => params[:ids]).all
+    @organizations = Organization.where(:id.in => params[:ids]).all
 
     @organizations.each do |org|
-      @employer_invoice = EmployerInvoice.new(org)
```

Comment:
As per David we should n't remove DC implementaion....we would like to still Prawn PDF for DC until we get approval from stakeholders. 

Add a condition here, to use old implemention for Dc and new engine for MA.